railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_8_0.rb.tt

```diff
@@ -14,8 +14,8 @@
 # If set to `:zone`, `to_time` methods will use the timezone of their receivers.
 # If set to `:offset`, `to_time` methods will use the UTC offset.
 # If `false`, `to_time` methods will convert to the local system UTC offset instead.
-#++
-# Rails.application.config.active_support.to_time_preserves_timezone = :zone
+# When you're ready to change, add this to `config/application.rb`
```

Comment:
In the past I've seen us use stronger language in this type of case, I think there is some nuance to be applied though:
https://github.com/rails/rails/blob/9c24bbb84f70dbc2ec9b909688eef11138cb0660/railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_1.rb.tt#L19

Reply:
I've copied this from:
https://github.com/rails/rails/blob/9c24bbb84f70dbc2ec9b909688eef11138cb0660/railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_1.rb.tt#L240-L242

The "(NOT this file)" part get lost somehow. I've added that again.

Reply:
There is also a note at the top of the blob:

https://github.com/rails/rails/blob/9c24bbb84f70dbc2ec9b909688eef11138cb0660/railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_1.rb.tt#L230